src/AutoRest.CSharp/Common/Input/InputTypes/Serialization/TypeSpecSerialization.cs

```diff
@@ -3,13 +3,65 @@
 
 using System;
 using System.Text.Json;
+using System.Text.Json.Nodes;
 using System.Text.Json.Serialization;
 using Azure.Core;
 
 namespace AutoRest.CSharp.Common.Input
 {
     internal static class TypeSpecSerialization
     {
+        /// <summary>
+        /// Since we need to parse the json using references, we require the "Properties" property in the json to appear the last after all other properties of the object.
+        /// This method here is a workaround that reads all the content of the json and reorders the properties so that "Properties" is the last property.
+        /// </summary>
+        /// <param name="json"></param>
+        /// <returns></returns>
+        internal static string NormalizeJson(string json)
```

Comment:
How is this function related to the LowConfidence features?

Reply:
this is only experimental